packages/assistant/src/RPA/Assistant/flet_client.py

```diff
@@ -169,20 +169,24 @@ def on_session_created(conn, session_data):
             # pylint: disable=raise-missing-from
             raise TimeoutException("Reached timeout while waiting for Assistant Dialog")
 
-    def _make_flet_event_handler(self, name: str):
+    def _make_flet_event_handler(self, name: str, handler: Optional[Callable] = None):
         def change_listener(e: ControlEvent):
+            handler(e)
             self.results[name] = e.data
-            e.page.update()
```

Comment:
Need to test that removing this doesn't break anything

Reply:
tested, I don't think it was required to begin with, because we don't need to re-render anything on-change